WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -169,11 +163,11 @@ class GutenbergViewController: UIViewController, PostEditor {
         return GutenbergMediaInserterHelper(post: post, gutenberg: gutenberg)
     }()
 
-    /// For autosaving - The debouncer will execute local saving every defined number of seconds.
-    /// In this case every 0.5 second
+    /// For autosaving - We are now using the Gutenberg side autosaving mechanism, so we don't need
+    /// a local scheduler any longer. We keep the debouncer to reuse the shared PostEditor+Publish save code.
     ///
     fileprivate(set) lazy var debouncer: Debouncer = {
-        return Debouncer(delay: PostEditorDebouncerConstants.autoSavingDelay, callback: debouncerCallback)
+        return Debouncer(delay: 0, callback: debouncerCallback)
```

Comment:
By using a `Debouncer` here with a zero delay we are still using a timer. I'm not sure if this would have any unintended side effects.

I think I'm overthinking this but I'll drop the suggestion just in case, what if this was a protocol and we could have a different implementation when we wanted a false debouncer that just runs immediately?
https://gist.github.com/koke/988287b7c7125af9b88f46ac6032e37a

Reply:
This change is not needed anymore 👍